docs/extras/nandrestore.md

```diff
@@ -1,36 +1,38 @@
-- The [Hekate](https://github.com/CTCaer/hekate/releases/) Payload
+- The <a href="https://github.com/CTCaer/hekate/releases/" target="_blank">Hekate</a> Payload
 - A microSD card larger than 32gb
 
 ### Instructions:
 
-Before we start, check if you have a tree of folders called `backup/[8 Character NAND id]/restore` on your sd card
+Before we start, check if you have a tree of folders called `backup/[8 Character NAND id]/restore` on your sd card.
 
 !!! warning "If you don't see a backup or [8 Character NAND id] folder on your sd card:"
-	1. Launch the latest hekate payload on your switch.
-	2. Navigate to `Tools > Backup > Backup eMMC BOOT0/1`, and press the power button to select the option.
+	This means you do not have a nand backup, it is highly reccomended you make one as soon as possible. Follow the steps below to make one.
+
+	1. Launch the latest Hekate payload on your switch.
+	2. Navigate to `Tools > Backup eMMC > eMMC BOOT0 & BOOT1` and let it do its thing.
 	3. Wait for the process to complete. When it's completed, you now have a `backup/[8 Character NAND id]/restore` folder on your sd card. Continue with step 1 of the guide.
 
-1. Copy your `Rawnand.bin` (combined or in 15 or 30 parts), `BOOT0`, and `BOOT1` to the `backup/[8 Character NAND id]/restore` folder on the sd card
-2. Put your sd card into your switch, and launch hekate.
-3. Navigate to `Tools > Restore`. Select `Restore eMMC BOOT0/1`. Wait for this process to complete. After it finished, you can press power to return.
-4. Select `Restore eMMC RAW GPP`. This WILL take a very long time to complete. After it finished, you can press power to return.
+1. Copy your `rawnand.bin` (combined or in 15 or 30 parts, this can be done by using the joiner scripts on the hekate github release page), `BOOT0`, and `BOOT1` to the 	`backup/[8 Character NAND id]/restore` folder on the sd card.
+2. Put your sd card into your switch, and launch Hekate.
+3. Navigate to `Tools > Restore eMMC`. Select `Restore eMMC BOOT0 & BOOT1`. Wait for this process to complete.
+4. In that same menu, select `eMMC RAW GPP` and wait for the process to complete. This will take a very long time to complete.
 
-!!! danger "If you're downgrading using your nand backup"
-	If the MAJOR version you were on before you performed the nand restore is HIGHER than the nand backup itself, you have to enable autoRCM to not get stuck in a boot crash.
-	Major versions include 1.0, 2.0, 3.0, 4.0, 4.1, 5.0, 6.0, 6.2, 7.0
+!!! danger "If you're downgrading using your NAND backup"
+	If the MAJOR version you were on before you performed the NAND restore is HIGHER than the NAND backup itself, you have to enable autoRCM to not get stuck in a boot crash.
+	A system update is considered a major version when a fuse is burned, you can check **<a href="https://switchbrew.org/wiki/Fuses#Anti-downgrade" target=blank>which versions burn fuses here</a>**.
```

Comment:
I don't like this wording. It interferes with semantic version naming, which Nintendo does more or less adhere to (`x.y.z`, MAJOR being `x`)